### ISSUE TITLE
Please provide example of how to get rod-stream working inside a docker image

### ISSUE THREAD

**Issue by bob9** [navicstein/rod-stream#13]
I have attempted to get this running inside a docker image with no success.  

I would like to provide a rest api which will scrape a page. Then upload the video to a remote location.

I was trying to use ghcr.io/go-rod/rod as the base image. However i think the main issue is the lack of X system running.  I have tried XVFB() with no luck any help here is also appreciated.

My current launcher looks like this.

```
	preferences := fmt.Sprintf(`{"download": {"dx/efault_directory": "%s"}}`, downloadDirectoryAbs)
		rodLauncher = rodstream.MustPrepareLauncher(rodstream.LauncherArgs{UserMode: false}).Preferences(preferences).Set("no-sandbox").HeadlessNew(true).Bin(path)

```

MustCreatePage is being called and I get this error "Failed to initialise stream for scraper video capture! Reason: videoCapturePage not created yet, call MustCreatePage" because if t.Type == proto.TargetTargetInfoTypeBackgroundPage && t.Title == "Video Streamer" { always fails to file the Video Streamer page .

Any help is appreciated.

If there was an example with a Dockerfile and a basic main.go I am sure I could get it going if it is at all possible.

Thanks

**Comment by mewforest** [comment]
I agree, it would be great, if author will create docker example for this extension 🙏🏻

**Comment by bob9** [comment]
> I agree, it would be great, if author will create docker example for this extension 🙏🏻

This worked for me. You need to copy this folder "/go/pkg/mod/github.com/navicstein" from your builder Docker image and then it seems to work.  I have not included my golang builder in the docker image example below. Its assuming a docker multistage build.

Dockerfile
```
FROM ghcr.io/go-rod/rod
WORKDIR /home/ubuntu

RUN apt-get update && \
      apt-get -y install sudo

#RUN apt update -y && sudo apt upgrade -y
RUN sudo apt install -y ffmpeg
RUN mkdir -p /go/pkg/mod/github.com/navicstein

COPY --from=builder /go/bin/go-app /home/ubuntu/go-app
COPY --from=builder /go/pkg/mod/github.com/navicstein /go/pkg/mod/github.com/navicstein

EXPOSE 8081

ENTRYPOINT ["/home/ubuntu/go-app"]
```
